app/stores/ada/index.js

```diff
@@ -28,7 +28,7 @@ export type AdaStoresMap = {
   addresses: AddressesStore,
   daedalusTransfer: DaedalusTransferStore,
   trezorConnect: TrezorConnectStore,
-  trezor: TrezorStore,
+  trezorSendAda: TrezorSendAdaStore,
```

Comment:
I would skip `Ada` from the name because it’s already part of `AdaStoresMap` and the other stores also don’t reference it e.g. `daedalusTransfer` is not `daedalusTransferAda`

Reply:
Initially, I tried with `trezorSend` then changed to `trezorSendAda` because `trezorSend` does not make such sense to me.

![image](https://user-images.githubusercontent.com/19986226/49707673-b307c380-fc6f-11e8-89d0-646ba5e394cd.png)

![image](https://user-images.githubusercontent.com/19986226/49707711-ed716080-fc6f-11e8-9c4e-5512b37e5e60.png)

will make `trezorSend` as you say.

Reply:
https://github.com/Emurgo/yoroifrontend/pull/144/commits/d7e3381e337199521184e4d30f1f7c9be1d43dc5

Reply:
https://github.com/Emurgo/yoroi-frontend/pull/144/commits/d7e3381e337199521184e4d30f1f7c9be1d43dc5